src/test/shell/bazel/testdata/pluto.git_log

```diff
@@ -1,4 +1,40 @@
-commit b87de9346bb1a4a3d4d2ab1a567b07c5d11a486a (HEAD -> master, tag: 1-build)
+commit a367b7e7f8f362697a079b7d23935bd079058af8 (HEAD -> master, tag: 3-subdir-bare)
+Author: John Doe <john@foo.com>
```

Comment:
I know you didn't start it, but I would have preferred email adresses in the rfc2606-reserved domains for fictional characters. (This also applies to other parts of that change.)

Reply:
I shall enter a new PR to fix those (since it will be a big diff but mostly noise). Thanks!